web/src/popover_menus.js

```diff
@@ -231,7 +231,9 @@ export function toggle_message_actions_menu(message) {
 }
 
 export function do_schedule_message(send_at_time) {
-    overlays.close_active_modal();
+    if (overlays.is_modal_open()) {
+        overlays.close_modal("send_later_modal");
+    }
```

Comment:
I guess in theory a different modal could be open, but I'm not sure that's actually possible. Might be worth a discussion in #frontend about whether we should have an API of the form `overlays.close_modal_if_open("send_later_modal")` for this sort of situation.

Reply:
Okay.